lib/webdrivers/chromedriver.rb

```diff
@@ -96,7 +104,9 @@ def download_url
                     normalize_version(required_version)
                   end
 
-        file_name = System.platform == 'win' || System.wsl? ? 'win32' : "#{System.platform}64"
+        archi = apple_m1_cpu? && apple_m1_compatible?(version) ? '_m1' : ''
```

Comment:
Can we please rename `archi` to `apple_arch` or `mac_arch` since it's specific to macOS/apple? Another option is to pull out the filename logic into a separate method to keep the complexity low.